src/Tests/RemoteUnitTestExecutor/AssertExtensions.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+
+namespace RemoteUnitTestExecutor
+{
+    public static class AssertExtensions
+    {
+        public static void ClearInvokedMethods(IList<MethodInvocationInfo> invokedMethods)
+        {
+            invokedMethods.Clear();
+            Assert.AreEqual<int>(0, invokedMethods.Count, "Invoked methods are not cleared.");
```

Comment:
>0, invokedMethods.Count [](http://example.com/codeflow?start=33&length=23)

How could it be possible for this to be not true? Wouldn't it indicate some kind of race condition that should be accounted for elsewhere?

Reply:
This is what is written in the original code and I didn't want to change it unless I have to.